mergify_engine/redis_utils.py

```diff
@@ -202,6 +213,7 @@ def cache(self) -> RedisCache:
             config.LEGACY_CACHE_URL,
             decode_responses=True,
             max_connections=self.cache_max_connections,
+            redis_connect_func=load_scripts,
```

Comment:
I don't think this is required anymore, since we drop the migration scripts, we don't need them here anymore.